src/dbus_api/pool.rs

```diff
@@ -65,15 +66,23 @@ fn create_filesystems(m: &MethodInfo<MTFn<TData>, TData>) -> MethodResult {
             .collect::<Vec<(&str, Option<Sectors>)>>(),
     );
 
-    let msg = match result {
-        Ok(ref infos) => {
-            let return_value = infos
+    let infos = match result {
+        Ok(created_set) => created_set.changed(),
+        Err(err) => {
+            let (rc, rs) = engine_to_dbus_err_tuple(&err);
+            return Ok(vec![return_message.append3(default_return, rc, rs)]);
+        }
+    };
+
+    let return_value = match infos {
+        Some(ref i) => {
```

Comment:
Try calling ```i``` something like ```newly_created_filesystems``` to help out with reading.

Reply:
Resolved.